### DISCUSSION TITLE
windows平台如何构建macos或者ios

### DISCUSSION THREAD

**Discussion by jmgaooo** [xmake-io/xmake#3096] [Q&A]
请问使用tbox在windows下开发的程序，能不能在window下构建成macos的可执行文件？
试过`xmake f -p macosx -a x86_64 -m release`命令但是报错了，好像是找不到xcode，那-p macosx参数只能在mac环境使用吗？

**Comment by waruqi** [comment]
不能，这跟 xmake 没啥关系，ios/macos 程序只能用 xcode 编译。

**Reply by jmgaooo** [reply]
好的，谢谢
